### PR TITLE
Document VReplication Source Throttling For Copy Table

### PR DESCRIPTION
**Pull request by mattlord**
This documents the work done in https://github.com/vitessio/vitess/pull/9923

I also used this one liner to move all long flags from single dash to double dash (single is deprecated in 14.0) in the files edited in the PR:
```
sed -e ':duit' -r -i '' -e 's/(.*[[:space:]]+[`]*)(-[a-z,A-Z]+[a-z,A-Z,-_`]+)([[:space:]]+.*)/\1-\2\3/g' -e 't duit' <path_to_file>
```

That gets most of them and a manual review can be done to ensure everything is good.

### CODE REVIEW

File: content/en/docs/14.0/reference/vreplication/flags.md [hunk 1]
```diff
@@ -22,16 +22,32 @@ The relay log buffers events on the target as they are received from the source.
...
+* You can see the number of waits and time spent waiting for MySQL on the source tablet by table (we do not have the workflow name on the source) as `<tablename>:waitForMySQL` in `VStreamerPhaseTiming` at the `/debug/vars` vttablet endpoint 
+
+#### vreplication_copy_phase_max_mysql_replication_lag
+
+**Type** integer\
+**Unit** seconds\
+**Default** 43200 (12 hours)\
+**Applicable on** source
+
+When copying the contents of a table we go through 1+ cycles of copy,catchup,fastforward in the copy phase. When preparing to copy a batch of rows (row streamer) we check the [Seconds_Behind_Source](https://dev.mysql.com/doc/refman/en/replication-administration-status.html) value on the source MySQL instance and wait for it to become less than or equal to this value before beginning. This helps to limit the impact of VReplication operations such as `MoveTables` on the source tablet (especially important if the source is a PRIMARY).
```
Comment (FancyFane) [inline content/en/docs/14.0/reference/vreplication/flags.md]: In the description here you mention this is an important value if the source is a Primary; however Primary sources may not have replication lag, unless there is a scenario I'm not considering here.

**Review by FancyFane — COMMENTED** [review]
Reviewing the changes I only had one question for the technical content.

File: content/en/docs/14.0/reference/vreplication/flags.md (hunk 1 above)
Comment (mattlord) [reply content/en/docs/14.0/reference/vreplication/flags.md]: Good point. I'll remove that parenthetical. Thanks!

Comment (mattlord) [reply content/en/docs/14.0/reference/vreplication/flags.md]: Done

**Review by FancyFane — APPROVED** [review]
lgtm

**Pull request merged**